app/components/primer/icon_button.erb

```diff
@@ -0,0 +1,4 @@
+<%= render Primer::Beta::BaseButton.new(**@system_arguments) do -%>
+  <%= render Primer::OcticonComponent.new(icon: @icon) %>
+  <%= render Primer::Alpha::Tooltip.new(**@tooltip_arguments) %>
```

Comment:
@jonrohan

Sorry I just caught that this tooltip element is nested inside the `Button`. This should be adjacent to the `Button`. Similar with `Button` component and `Link` component.

This current nesting will result in the button (or link) being triggered when one tries to copy the tooltip text.

I think that in the case of tooltip, there will need to be a wrapper `div` or `span` element over the component and tooltip.